### PR TITLE
Add documentation for latesthttpresponse in Send External Object action

### PR DESCRIPTION
**Pull request by osoykan**

### CODE REVIEW

File: content/refguide/send-external-object.md [hunk 1]
```diff
@@ -27,6 +27,21 @@ This activity can only send objects from an OData service.
 
 Some services may have updatable attributes that are part of the key of the entity. Services published by Mendix apps do not have this, but other services might. It's not recommended to change those attributes.
 
+## 3 Response Handling
+
+### 3.1 $latestHttpResponse Variable
+
+The `$latestHttpResponse` variable is of the [HttpResponse](http-request-and-response-entities#http-response) type. It is available after a **Send External Object** activity.
+
+However, its `Content` attribute will be left empty in most cases to minimize memory usage.
```
Comment (joostverhoog) [inline content/refguide/send-external-object.md]: Can we specify in which cases it's empty?

File: content/refguide/send-external-object.md [hunk 2]
```diff
@@ -27,6 +27,21 @@ This activity can only send objects from an OData service.
 
 Some services may have updatable attributes that are part of the key of the entity. Services published by Mendix apps do not have this, but other services might. It's not recommended to change those attributes.
 
+## 3 Response Handling
+
+### 3.1 $latestHttpResponse Variable
+
+The `$latestHttpResponse` variable is of the [HttpResponse](http-request-and-response-entities#http-response) type. It is available after a **Send External Object** activity.
```
Comment (joostverhoog) [inline content/refguide/send-external-object.md]: It's also available after the Delete External Object activity, right? Shouldn't we add documentation there?

File: content/refguide/send-external-object.md [hunk 3]
```diff
@@ -27,6 +27,21 @@ This activity can only send objects from an OData service.
 
 Some services may have updatable attributes that are part of the key of the entity. Services published by Mendix apps do not have this, but other services might. It's not recommended to change those attributes.
 
+## 3 Response Handling
```
Comment (joostverhoog) [inline content/refguide/send-external-object.md]: It looks like this document repeats number 3 for Response Handling and Related now

File: content/refguide/send-external-object.md [hunk 4]
```diff
@@ -27,6 +27,21 @@ This activity can only send objects from an OData service.
...
+
+### 3.1 $latestHttpResponse Variable
+
+The `$latestHttpResponse` variable is of the [HttpResponse](http-request-and-response-entities#http-response) type. It is available after a **Send External Object** activity.
+
+However, its `Content` attribute will be left empty in most cases to minimize memory usage.
+
+This attribute is filled when one of the following scenarios occur:
+
+* The **Response handling** is **Store in an HTTP response** and the call succeeded
```
Comment (joostverhoog) [inline content/refguide/send-external-object.md]: Is this part of the Studio Pro UI now?

**Review by joostverhoog — CHANGES_REQUESTED** [review]

File: content/refguide/send-external-object.md (hunk 1 above)
Comment (osoykan) [reply content/refguide/send-external-object.md]: I've checked it again to validate, it is always filled, so I'm deleting this part.

File: content/refguide/send-external-object.md (hunk 4 above)
Comment (osoykan) [reply content/refguide/send-external-object.md]: removed

File: content/refguide/send-external-object.md (hunk 3 above)
Comment (osoykan) [reply content/refguide/send-external-object.md]: changed

File: content/refguide/send-external-object.md (hunk 2 above)
Comment (osoykan) [reply content/refguide/send-external-object.md]: added

File: content/refguide/send-external-object.md [hunk 5]
```diff
@@ -27,7 +27,15 @@ This activity can only send objects from an OData service.
 
 Some services may have updatable attributes that are part of the key of the entity. Services published by Mendix apps do not have this, but other services might. It's not recommended to change those attributes.
 
-## 3 Related
+## 3 Response Handling
+
+### 3.1 $latestHttpResponse Variable
+
+The `$latestHttpResponse` variable is of the [HttpResponse](http-request-and-response-entities#http-response) type. It is available after **Send External Object**, **Delete External Object** activities.
```
Comment (joostverhoog) [inline content/refguide/send-external-object.md]: I expected a change in the page for delete external object as well, and no mention of that activity here.

File: content/refguide/send-external-object.md [hunk 6]
```diff
@@ -27,7 +27,15 @@ This activity can only send objects from an OData service.
...
+
+This variable can be accessed from any microflow action in the scope.
```
Comment (joostverhoog) [inline content/refguide/send-external-object.md]: Isn't it called a microflow _activity_ rather than a microflow _action_?

**Review by joostverhoog — CHANGES_REQUESTED** [review]

File: content/refguide/send-external-object.md (hunk 5 above)
Comment (osoykan) [reply content/refguide/send-external-object.md]: I see. But there is no page for delete external object as far as I searched in the docs.

File: content/refguide/send-external-object.md (hunk 6 above)
Comment (osoykan) [reply content/refguide/send-external-object.md]: addressed

File: content/refguide/send-external-object.md (hunk 5 above)
Comment (osoykan) [reply content/refguide/send-external-object.md]: Also removed `Delete External Object` addition from here.

Comment (johannahemminger) [reply content/refguide/send-external-object.md]: https://docs.mendix.com/refguide/delete-external-object/ - do you mean this one?

**Comment by osoykan** [comment]
Please refer this PR: https://github.com/mendix/docs/pull/4596, this one is based on outdated development

**Pull request closed without merging**